osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyCalculator.cs

```diff
@@ -74,18 +71,16 @@ protected override IEnumerable<DifficultyHitObject> CreateDifficultyHitObjects(I
             }
         }
 
-        protected override Skill[] CreateSkills(IBeatmap beatmap) => new Skill[]
-        {
-            new Aim(),
-            new Speed()
+        protected override Skill[] CreateSkills(IBeatmap beatmap) => new Skill[] {
+            new Aim (),
+            new Speed ()
         };
 
-        protected override Mod[] DifficultyAdjustmentMods => new Mod[]
-        {
-            new OsuModDoubleTime(),
-            new OsuModHalfTime(),
-            new OsuModEasy(),
-            new OsuModHardRock(),
+        protected override Mod[] DifficultyAdjustmentMods => new Mod[] {
+            new OsuModDoubleTime (),
+            new OsuModHalfTime (),
+            new OsuModEasy (),
+            new OsuModHardRock (),
         };
```

Comment:
We do not put spaces before parentheses. Please adhere to our code style and run style inspection using the provided `InspectCode.ps1` script.

Reply:
Fixed in 0cb6464.